group/iab/IAB_Decisions.md

```diff
@@ -54,56 +59,30 @@ for the IAB chair to determine whether it will be held open or secret.
-  irregularly scheduled meeting (i.e. meetings other than the weekly IAB
-  meeting teleconference or meetings prescheduled during IETF week) then, the
-  new meeting needs to be announced at least a week in advance.
-
-- Discussion and votes will continue on the voting members list, and during the
-  teleconference.
-
-- The vote will typically stay open until a vote has been recorded for every
+typically follow this process to initiate a vote:
+
+- The chair emails the list of voting members to open the vote and start an email discussion on
+  the topic, optionally indicating that the ballots will be secret.
+
+- Discussion and votes will continue on the voting members list, and during any
+  teleconference(s) that occur while the vote is open.
+
+- The vote will typically stay open until a position has been recorded for every
   single voting member, even after a majority has been reached. This ensures
   every voice can be heard. Note that it is reasonable for any voting member to
   request a few days to review available information before voting.
 
-- If the vote has enough affirmative votes that it would pass if all remaining
-  votes were dissents, and at least 7 days have elapsed since the
-  teleconference, then the chair can decide to declare the vote closed and
-  successful. This avoids deadlock if a voting member is unable or unwilling to
-  vote.
-
-- A vote can result as a consequence of a teleconference discussion (e.g., the
-  topic of an appointment was on the agenda of the teleconference without
-  explicitly mentioning a vote, but during the discussion there was consensus
-  to start a vote on the topic). In those cases, votes can be tallied in the
-  teleconference, and/or the email to announce the vote can be sent out without
-  requiring a second teleconference on this topic.
-
-## Expedient Decisions
-
-If a decision is needed within a short time frame, any IAB member (or the
-executive director) can call a vote without first placing the topic on an IAB
-meeting agenda. If all voting members reply and they have all voted
-affirmatively or recused themselves, then the decision is considered to have
-the full consensus of the IAB and does not need to follow the Normal Process
-described above. If any voting member votes against the decision, has not
-responded, or simply requests a discussion on the topic, then the topic is
-added to the agenda of an upcoming IAB meeting and the Normal or Alternative
-Process is followed as usual.
+- At the chair's option, a vote can be held open until there is an opportunity
+  to discuss it in the next meeting. Likewise, if more than two voting members
+  request discussion in the next meeting, the vote will remain open until that
+  discussion takes place. Here, 'next meeting' is the next regularly scheduled
+  meeting more than 48 hours after the first request is received, or in the
+  case of an irregularly scheduled meeting, 7 days after that time.
```

Comment:
```suggestion
```

I don't think this is needed. Two reasons:
- If this is at the start, instead of holding the vote open until a discussion can happen in a teleconference, the chair could just introduce the topic and say that the vote will happen at the meeting.
- If this is a result of discussion going on on the topic, any more than two members want to have more discussion, the vote cannot conclude successfully anyhow.

Reply:
I'm suggesting a replacement paragraph at the end that explains that 3 or more voting members can decide to defer the conclusion of the vote.